docs/infra/api-markdown-documenter/api-documentation-layout.mjs

```diff
@@ -28,9 +28,9 @@ import { AdmonitionNode } from "./admonition-node.mjs";
 const customExamplesSectionTitle = "Usage";
 const customThrowsSectionTitle = "Error Handling";
 
-const supportDocsLinkSpan = new SpanNode([
+const supportDocsLinkParagraph = new ParagraphNode([
```

Comment:
Note for reviewers: Directives in `mdast` require block content children. All of the spans here have been updated to be paragraphs.